exercises/wordy/wordy_spec.rb

```diff
@@ -0,0 +1,136 @@
+require_relative "./wordy"
+
+RSpec.describe WordProblem do
+  it "can add one to one" do
+    problem = WordProblem.new("What is 1 plus 1?")
+
+    expect(problem.answer).to eq(2)
+  end
+
+  it "can add two to fifty three" do
+    skip "fix this test second - delete the `skip` lines to enable individual tests"
```

Comment:
I think `skipped` is what we're looking for. It should always be skipped, until they delete it. It's not temporarily skipped because we think we have a bug.

Is there a way to not have the verbose output about it being skipped? I'm trying to avoid noisy output. In this context (exercism) it's not intended as team-wide communication.